### ISSUE TITLE
name 'MeshLambertMaterial' is not defined

### ISSUE THREAD

**Issue by mznyc** [jovyan/pythreejs#136]
## running the following example with error

from pythreejs import *
from IPython.display import display
from ipywidgets import HTML, Text
from traitlets import link, dlink
ball = Mesh(geometry=SphereGeometry(radius=1), 
            material=MeshLambertMaterial(color='red'),
            position=[2, 1, 0])

c = PerspectiveCamera(position=[0, 5, 5], up=[0, 1, 0],
                      children=[DirectionalLight(color='white', position=[3, 5, 1], intensity=0.5)])

scene = Scene(children=[ball, c, AmbientLight(color='#777777')])

renderer = Renderer(camera=c, 
                    scene=scene, 
                    controls=[OrbitControls(controlling=c)])
display(renderer)

NameError                                 Traceback (most recent call last)
<ipython-input-6-069771b62a9d> in <module>()
      4 from traitlets import link, dlink
      5 ball = Mesh(geometry=SphereGeometry(radius=1), 
----> 6             material=MeshLambertMaterial(color='red'),
      7             position=[2, 1, 0])
      8 

NameError: name 'MeshLambertMaterial' is not defined

**Comment by vidartf** [comment]
Note: The examples in the repo are incompatible with the latest stable release. For examples related to the stable release, see https://github.com/jovyan/pythreejs/tree/0.4.x/examples.

**Comment by vidartf** [comment]
If this issue is unrelated to that, could you please say which version you have installed, and link to the example you are trying to run?

**Comment by mznyc** [comment]
I was running it from the master branch here 
https://github.com/jovyan/pythreejs/blob/master/examples/Examples.ipynb

It seems the stable release version does not have the problem 
 https://github.com/jovyan/pythreejs/tree/0.4.x/examples

**Comment by mznyc** [comment]
However I do not see any figure shown after running the script

ball = Mesh(geometry=SphereGeometry(radius=1), 
                                    material=LambertMaterial(color='red'),
                                    position=[2, 1, 0])

scene = Scene(children=[ball, AmbientLight(color='#777777')])

c = PerspectiveCamera(position=[0, 5, 5], up=[0, 0, 1],
                      children=[DirectionalLight(color='white', 
                                                 position=[3, 5, 1], 
                                                 intensity=0.5)])
renderer = Renderer(camera=c, 
                    scene=scene, 
                    controls=[OrbitControls(controlling=c)])
display(renderer)

**Comment by vidartf** [comment]
Try to upgrade pythreejs to 0.4.1 for the 0.4.x examples. Then, check both the browser console, and the notebook server log for any errors.

**Comment by vidartf** [comment]
@mznyc Are you still having issues with this? Did you try updating to 0.4?

**Comment by mznyc** [comment]
@vidartf Thanks for the follow-up. I have yet to do that. Will update once I give it try.

**Comment by mznyc** [comment]
I have upgraded pythreejs to 0.4.1. Still no plots.

**Comment by vidartf** [comment]
Could you run the code again, and copy any output from the developer console in the browser (and also check the notebook server log for any exceptions)?

**Comment by rsignell-usgs** [comment]
I did `conda install -c conda-forge pythreejs` and I'm at version 0.4.1.

I'm getting the same problems with the notebook
https://github.com/jovyan/pythreejs/blob/master/examples/Examples.ipynb
```
      1 ball = Mesh(geometry=SphereGeometry(radius=1), 
----> 2             material=MeshLambertMaterial(color='red'),
      3             position=[2, 1, 0])
      4 
      5 c = PerspectiveCamera(position=[0, 5, 5], up=[0, 1, 0],

NameError: name 'MeshLambertMaterial' is not defined
```

Is this expected?

**Comment by psychemedia** [comment]
I am having similar issues. Executable version via *mybinder* here: https://beta.mybinder.org/v2/gh/psychemedia/showntell/d3

Configuration here: https://github.com/psychemedia/showntell/tree/3d

**Comment by vidartf** [comment]
@rsignell-usgs The example you were trying to run is tied to the unreleased code in master, in which several classes have been renamed. Please check out the examples in the 0.4.x branch in the mean time. See my initial response to the issue report for a link.

**Comment by rsignell-usgs** [comment]
I finally got some examples running on Windows.

TL;DR:
```
conda install -c conda-forge pythreejs
git clone https://github.com/jovyan/pythreejs.git --branch 0.4.0
```

Long version: I first tried to pip install master, but installation failed (couldn't copy/move some file, possibly a windows/linux path issue).

I then did `conda install -c conda-forge pythreejs`, which installed version 0.4.1, based on pypi 0.4.1 package.

Then I tried to figure out what examples corresponded to this release.  This was harder because the pypi tar.gz package has stripped the examples, and there was no release tagged 0.4.1 on github.  :cry: 

So I cloned the repo and tried checking out hashes that were close to the Oct 27 date that the pypi package was made, but the examples still didn't work. 

I ended up reverting back to the 0.4.0 release, which has only 2 examples.   But at least it let's you see something on windows:

![2017-11-23_11-30-04](https://user-images.githubusercontent.com/1872600/33183693-daafabb0-d046-11e7-9517-753a92d549e9.png)

**Comment by vidartf** [comment]
@psychemedia The binder repo you linked seem to work now. Are you still having issues?

**Comment by psychemedia** [comment]
I think I fixed it by changing the versions of either the package or the code I was using.. Thx...

**Comment by holdenweb** [comment]
Not looking to reopen this issue, but suggest it would be helpful to adopt user acceptance testing to verify examples work (or at least run without raising) for each release. Ended up here because Python couldn't find `Mesh`. Non-functioning example code doesn't give a good first impression, and I'm still unsure as to where I might find working example code. This is a great and worthy project, so I'm persisting :-).

**Comment by vidartf** [comment]
@holdenweb The issue here was that we merged into master before it was fully ready for release, and as such the new and updated examples no longer work with the latest release (they work with the beta). The release is right around the corner, so this will soon sort itself out. There is a note in progress in the documentation on breaking changes (probably more a short-list than a full changelog, given the scale of the changes).

**Comment by vidartf** [comment]
With 1.0.0 released, I'm going to close this as resolved.
